sw/device/lib/dif/dif_otbn.c

```diff
@@ -199,9 +199,11 @@ dif_result_t dif_otbn_set_ctrl_software_errs_fatal(const dif_otbn_t *otbn,
 }
 
 size_t dif_otbn_get_dmem_size_bytes(const dif_otbn_t *otbn) {
+  OT_DISCARD(otbn);
```

Comment:
maybe it would make more sense to simply remove this function altogether, and just use the macro `OTBN_DMEM_SIZE_BYTES` at the call sites instead?

Same for the `dif_otbn_get_imem_size_bytes()` function.

Reply:
Sure, replaced with the constant.